.github/FUNDING.yml

```diff
@@ -1,15 +1,15 @@
 # These are supported funding model platforms
 
 github: Dishpit
```

Comment:
i only have and use this currently; i've got others, but i don't use them much so the rest can stay blank for now

Reply:
sorted